scripts/release_notes/release_notes.py

```diff
@@ -156,21 +156,23 @@ def extract_notes(commit, seen):
 
     result = {}
 
-    notes = ""
+    notes = message
```

Comment:
This is confusing, I understand you did this because `extract_notes_from_rebase_commit` seemingly returns notes but it returns a messages, not yet notes, the message should becomes notes after the regex. Can we rename `extract_notes_from_rebase_commit` to `extract_message_from_rebase_commit` or something?

Reply:
Can we agree that this is not really important and to circle back to it later once this is merged?

Reply:
What for? We're already introducing a change here, this is a 1min change, not worth pushing it back to an hypothetical later change.

Reply:
And it is important, it's currently wrong

Reply:
I can do it if you want

Reply:
alright, I reverted it